NEWS.md

```diff
@@ -28,6 +28,9 @@ OpenSSL 3.2
     by default.
   * TCP Fast Open (RFC7413) support is available on Linux, macOS, and FreeBSD
     where enabled and supported.
+  * Full support for provider-based/pluggable signature algorithms in TLS 1.3
+    operations as well as X.509 data structure support. With a suitable provider
+    this fully enables use of post-quantum/quantum-safe cryptography.
```

Comment:
These used to be one-liners 🤣

Reply:
Sorry, didn't know that. What about changing this to the one-liner originally put into the CHANGES.md?

Reply:
Sure!

And just to be clear, my laughter wasn't specifically directed at you.  The NEWS section for the next version has no one-liners at all, I found that comical.

Reply:
Well, OK -- then I prefer to keep as-is (in line with the other statements (sizes)).